add cni deep dive part 1 to blog (#59)
cilium/cilium.io: src/posts/2022-02-23-cni-deep-dive/index.md

---
date: '2022-02-23T17:00:00.000Z'
title: 'Cilium Kubernetes CNI Provider Deep Dive: Part 1'

> ogImageUrl: 'https://heise.cloudimg.io/v7/_www-heise-de_/imgs/18/2/9/5/4/2/1/8/cilium-diagramm-da030a781be5edc2.png?org_if_sml=1&q=70&width=1220'

Author: ```suggestion
ogImageUrl: 'https://docs.cilium.io/en/stable/_images/cilium-arch.png'
```

> ogSummary: 'This video covers what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'

Author: ```suggestion
ogSummary: 'This video from The Learning Channel covers what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'
```

> ogSummary: 'Learn what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'

Author: ```suggestion
ogSummary: 'This video covers what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'
```
